shared/elements/gaia_grid/js/grid_view.js

```diff
@@ -544,13 +544,13 @@
           item.setCoordinates(xPosition, this.layout.offsetY);
           if (!item.active) {
             item.render();
+          }
 
-            if (item.detail.type === 'divider') {
-              if (oddDivider) {
-                item.element.classList.add('odd');
-              } else {
-                item.element.classList.remove('odd');
-              }
+          if (item.detail.type === 'divider') {
```

Comment:
if you set oddDivider to null in the initialization you can check here if (oddDivider !== null) instead of item.detail.type === 'divider' because it is a bit slower. if oddDivider is true or false means this is a divider. But just an idea, do it only if you consider good idea